client/signup/steps/new-or-existing-site/index.tsx

```diff
@@ -31,6 +31,7 @@ export default function NewOrExistingSiteStep( props: Props ): React.ReactNode {
 				{ stepName: props.stepName },
 				{
 					newOrExistingSiteChoice: value,
+					shouldHideFreePlan: true,
```

Comment:
Setting this to `true` enables auto-generation of the wordpress.com subdomain:
https://github.com/Automattic/wp-calypso/blob/trunk/client/lib/signup/step-actions/index.js#L138

Reply:
That's a good suggestion and totally makes sense.